tests/test_views.py

```diff
@@ -333,3 +333,51 @@ def test_it_should_ignore_token_type(self):
         res = self.view_post(data={'token': str(token)})
         self.assertEqual(res.status_code, 200)
         self.assertEqual(len(res.data), 0)
+
+
+class TestTokenBlacklistView(APIViewTestCase):
```

Comment:
This test should be testing that the token can't be used again, even if it is valid.

Reply:
Thanks, for the review! I have added the test, but I have not got the time to properly enable the base testcase to make calls to other endpoints than defined in `view_name`, so I have made a workaround. I hope you do not mind!